ckanext/geocat/harvester.py

```diff
@@ -1,45 +1,54 @@
 # -*- coding: utf-8 -*-
```

Comment:
We can remove these lines from all files now - they are only needed by Python 3 if we are using an encoding different from UTF-8, which afaik we never do.